knife-zero.gemspec

```diff
@@ -28,5 +28,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "simplecov"
   spec.add_development_dependency "simplecov-rcov"
 
-  spec.add_runtime_dependency "chef", ">= 12.6", "< 15.0"
+  spec.add_runtime_dependency "chef", ">= 12.6", "< 16.0"
```

Comment:
__[RuboCop]__ Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. ([link](https://sider.review/gh/repos/23179315/pulls/123#issue-4081909))

<sub>posted by Sider</sub>